plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/global-styles/variation-container.jsx

```diff
@@ -57,10 +57,7 @@ export const VariationContainer = ( { variation, children } ) => {
 					user.settings,
 					variation.settings
 				),
-				styles: mergeBaseAndUserConfigs(
-					user.styles,
-					variation.styles
-				),
+				styles: variation.styles,
```

Comment:
Hey @albarin Thanks for working on this! 👍 

Unlike the GB site editor, I use `mergeBaseAndUserConfigs` here because we've split style variations into font and color parts. We need to preserve the color changes when updating the font and vice versa.

I tested this PR on JN, and you can see when I change the color and then update the font, the color gets reset.  I think this is a bit tricky. Perhaps a way to fix it is to create a custom merge function. 🤔 

https://github.com/woocommerce/woocommerce/assets/4344253/dcc11241-8054-40f4-bb7c-277d30699620

Reply:
Thanks @chihsuan, great feedback!
I've addressed it https://github.com/woocommerce/woocommerce/pull/44424/commits/15a1b4e85bd24ae5ead254ed7cabaaf18b68cf25 let me know what you think.
What I'm doing it remove all the typhography settings from the styles before applying new ones.